INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# Installing the eZ Platform legacy bridge
+
+Unlike eZ Publish 5.x, eZ Platform does not include the Legacy stack by default.i
+Even though it is not officially supported, eZ Publish Legacy can easily be installed
+on top of Platform using Composer.
+
+### Missing legacy extensions
+
+Several ezpublish-legacy extensions are no longer installed by default, such as ezfind.
+They can still be manually added to the root `composer.json`.
+ 
+### Add the composer `legacy post-*-scripts`
+
+Edit `composer.json`, and add those lines to both `post-update-cmd` and `post-install-cmd` blocks at the end:
+
+```
+"scripts": {
+    "post-install-cmd": [
+        ...,
+        "eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installAssets",
+        "eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installLegacyBundlesExtensions"
+    ],
+    "post-update-cmd": [
+        ...,
+        "eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installAssets",
+        "eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installLegacyBundlesExtensions"
+    ],
+}
+```
+
+### Enable EzPublishLegacyBundle
+Edit `app/AppKernel.php` (or `ezpublish/EzPublishKernel` before ezplatform 2015.12),
+and add `new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle( $this )` at the end of 
+the `$bundles` array. Pay close attention to the `$this` argument. The LegacyBundle is a bit 
+of a spoiled brat, and has high expectations from its collaborators.
+
+### Add legacy routes
+Edit `app/config/routing.yml`, and add the LegacyBundle routes.
+
+```
+_ezpublishLegacyRoutes:
+    resource: @EzPublishLegacyBundle/Resources/config/routing.yml
+```
+
+### Enable legacy_mode for your backoffice siteaccess
+
+The Legacy Backoffice requires the `legacy_mode` option to be enabled.
+This can be done in app/config/config.yml or app/config/ezplatform.yml:
+
+```
+ez_publish_legacy:
+    system:
+        site_admin:
+           legacy_mode: true
+```
+
+### Optional: add security rules for the Setup Wizard
+
+If you intend to run the legacy setup wizard, you need to allow it in `app/config/security.yml`.
+
+```
+ezpublish_setup:
+    pattern: ^/ezsetup
+    security: false
+```
+
+### Install `ezsystems/legacy-bridge`
```

Comment:
Before this step, it should be noted that `"ezpublish-legacy-dir" "ezpublish_legacy"` needs to be added to `extra` section in `composer.json`. Otherwise, legacy install will end up in the root folder of the installation.